src/olympia/pages/templates/pages/shield_study_8.html

```diff
@@ -0,0 +1,62 @@
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Search Shield Study</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/unified-urlbar-shield-study/?src=shield-study-8"><button class="button primary large">Try Search Shield Study</button></a>
+        </div>
+        <div class="content"><h1 id="optimizing-firefox-search">Optimizing Firefox Search</h1>
+<p>Mozilla wants to know if we can improve the default search experience in Firefox. Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p>** If you agree to participate:</p>
+<ol>
+<li>We will add additional features to Firefox that modify the default search experience</li>
+<li>Continue using Firefox as you normally would. You may not notice any changes immediately.</li>
+<li>After 30 days, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>We will ask you to fill out a quick optional survey to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own in 30 days. You may leave the study early. To do so, follow these steps:</p>
+<ol>
+<li>Type about:addons into the location bar, and press enter.</li>
+<li>Find Search Shield Study in the addons list.</li>
+<li>Click the remove button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<h3 id="all-shield-studies">All Shield Studies</h3>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with Firefox <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Telemetry collection</a>.</p>
+<h3 id="this-particular-study">This particular Study</h3>
+<p>In addition to the data collected by all Shield Studies, here are the key things you should know about what is happening when you participate in this study:</p>
+<ul>
+<li>We will track basic information about your search behavior in Firefox. This includes using the in-product search tools and information about the search providers you may visit.</li>
+<li>No actual search queries will be collected. We will look at numbers of searches performed.</li>
+<li>No information will be collected while in Private Browsing mode</li>
+</ul>
+<p>You can learn more about the data collection for this study <a href="https://github.com/mak77/unified-urlbar-shield-study/blob/master/docs/metrics.md">here</a>.</p>
```

Comment:
Friendly reminder that "here" is not a very good description for a link. Can we change and update that for accessibility reasons and describe the target of the link a bit more?